pkg/kubectl/describe.go

```diff
@@ -1447,7 +1447,7 @@ type HorizontalPodAutoscalerDescriber struct {
 }
 
 func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (string, error) {
-	hpa, err := d.client.Extensions().HorizontalPodAutoscalers(namespace).Get(name)
```

Comment:
API server 1.2 supports kubectl 1.1 by not removing the endpoints. 
Kubectl 1.2 supports API server 1.1 by doing version negotiation and using 1.1 endpoints/versions/groups.